navigation bar moving effect removed and link visible- issue#33,issue#31

AshishkrGoyal opened the pull request:
@mariobehling  sir please review this pull request..
Thanks 😄

isuruAb commented:
@AshishkrGoyal  please give us a preview link. because it is hard to review style changes like this.
cheers

AshishkrGoyal commented:
@isuruAb 
please visit
 [http://www.ashishkrgoyal.me/2018.fossasia.org/](url) 
for review the changes.

#31 
#33 

both solved in this PR.

Thanks

niccokunzmann commented:
@AshishkrGoyal I am a bit confused and not expert. So, it would help, if the PR only solves one issue. So that I can understand it hopefully. If you could create one for only removing the moving effect, I would be glad to merge it. The button text is white now. I do not like to decide which one is merged because that means I need to know a lot. If all people working on one issue can coordinate, this is a big help to maintainers. The people working on the issue should have the most expertise and decide what is right.

niccokunzmann commented:
@AshishkrGoyal I would like to merge your PR, too. Now it has conflicts. Can you remove them?

niccokunzmann commented on .idea/workspace.xml:
```diff
@@ -0,0 +1,242 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Could you remove this file and add `/.idea` to the `.gitignore` file? This is not needed for the website.

AshishkrGoyal replied:
Sure.. Will do it

niccokunzmann commented:
In the preview, I see that you removed the moving effect. I believe, this is a solution.

AshishkrGoyal commented:
@niccokunzmann 
please merge it now, all conflicts has been removed 😄 

Thanks

niccokunzmann commented on index.html:
```diff
@@ -61,15 +61,6 @@
 							<div class="col-md-10 text-right">
 								<ul class="menu">
 									<li><a target="default" class="inner-link" href="#top">Home</a></li>
-									<!-- <li><a target="default" class="inner-link" href="#keynotes">Keynotes</a></li>
```
This may be needed later on in the process. I think, we can use this, once the conference organization has more information. Please let the comment stay in there.

niccokunzmann replied:
Generally, I understand why to remove comments but this code is sure to be needed next year. What do you think, should it be removed or could you live with the comment?

AshishkrGoyal replied:
ohh.. will do it ASAP!!
My bad

AshishkrGoyal replied:
live with the comments, i have no problems with this..
i will commit again .

AshishkrGoyal commented:
@niccokunzmann 
please check it now..
i have added all the comments now.

niccokunzmann commented on index.html:
```diff
@@ -1,382 +1,382 @@
-							</div>
+                        <div class="col-md-4 col-sm-6 sponsor-column">
+                            <div class="sponsor text-center">
+                                <a href="http://intel.com" target="default">
```
You are a skilled programmer, I do not doubt that. However, now I see a huge diff for the index file which I am not going to agree to because I like to follow the [CCCC](https://rfc.zeromq.org/spec:22/C4/).

> A patch SHOULD be a minimal and accurate answer to exactly one identified and agreed problem.

The changing the spaces is not an agreed problem.
The patch is not a minimal solution to **one** problem.

There are two solutions for this:
1. I merge and revert.
2. You change the patch.

Now, because I value what you do, I will tell you about my reasoning behind following CCCC:
Building open-source software is not about building software right but about building the right software.
In order to build the right software, we need a lot of ideas from as many people as possible, including you. However, there is a tradeoff. If a change scares away more people than it brings to the software, the diversity and future of the project is at risk of decline.

Thus, I am not against any changes you make. I will gladly accept any changes as long as I think they go by the rules I follow. In your case, 2-3 PRs would do it, I guess.
Also, I am not a gatekeeper. If you find any other person with access rights to merge this, they can do it.

What do you think about this?
How would you like to proceed?

AshishkrGoyal replied:
@niccokunzmann 
please help me with it..
what i have to do for moving further??

thanks in advance

AshishkrGoyal commented:
@isuruAb Sir..
please help me here ..
because at this point i m stuck

thanks in advance

isuruAb commented:
@AshishkrGoyal  it is pleasure to see good contribution from you. code aligning is good job and it increases readability. but now it is hard to see what you have actually edited.  please send only the changes that related to question.

AshishkrGoyal commented:
@isuruAb Sir..
here i have changed just custom.css file and first unfortunately i deleted comments then after i added them again so you are seeing changes in index.html also..
but in actually i have just changed custom.css file 

@isuruAb Sir..
i will commit again with very carefully then please review it ASAP 

Thanks for helping Sir 😄

isuruAb commented:
@AshishkrGoyal  always follow best practices then you can avoid this kind of problems. read this to know more about best practices:https://blog.fossasia.org/open-source-developer-guide-and-best-practices-at-fossasia/
happy coding !👍

niccokunzmann commented:
@AshishkrGoyal In order to start anew without the changes you have now, you can do the following:

    git checkout ccc666be4249d2db31e971e2d3c8db4b62bea18c
    git checkout -b fix-issue-33

Now, you can change things in the fix-issue-33 branch

    git commit -am"fix issue #33"
    git push -u origin fix-issue-33

Visit https://github.com/fossasia/2018.fossasia.org/compare/master...AshishkrGoyal:fix-issue-33 to create another pull-request.

AshishkrGoyal commented:
@isuruAb @niccokunzmann 
please check #42  for this 

Thanks

niccokunzmann commented:
@AshishkrGoyal I will close this because of PR #42. We can still re-open it it needed.
